src/array.jl

```diff
@@ -76,6 +89,9 @@ macro implement_array_methods(t)
         function Base.copyto!(dest::PermutedDimsArray{T,N}, src::$t{T,N}) where {T,N}
             return $_disk_copyto!(dest, src)
         end
+        function Base.copyto!(dest::Vector{T}, dstart::Integer, src::$t{T, 1}, sstart::Integer, n::Integer) where {T}
```

Comment:
Is `T` always identical for scr and dest in Bae `copyto!` ? What if you have different Real types

Reply:
We also use `setindex!` when wtiting into the chunk buffer, so it should be the same?